.github/workflows/main.yml

```diff
@@ -38,8 +42,15 @@ jobs:
           MYSQL_ROOT_PASSWORD: root
           MYSQL_DATABASE: test
     steps:
+
     - name: Checkout code
       uses: actions/checkout@v1
+    - name: .NET setup
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: |
+          3.1.x
+          5.0.x
```

Comment:
Dapper needs a lot of updating love we haven't had time for - this _shouldn't_ be needed, but needs to be paired with a net6.0 update _on test projects only_ (not changing core library targets). We can do that in this PR or separately.